meilisearch-http/tests/index/create_index.rs

```diff
@@ -36,6 +40,44 @@ async fn create_index_with_gzip_encoded_request() {
     assert_eq!(response["details"]["primaryKey"], Value::Null);
 }
 
+#[actix_rt::test]
+async fn create_index_with_gzip_encoded_request_and_receiving_brotli_encoded_response() {
+    let server = Server::new().await;
+    let app = test::init_service(create_app!(
+        &server.service.meilisearch,
+        &server.service.auth,
+        true,
+        server.service.options,
+        analytics::MockAnalytics::new(&server.service.options).0
+    ))
+    .await;
+
+    let body = serde_json::to_string(&json!({
+        "uid": "test",
+        "primaryKey": None::<&str>,
+    }))
+    .unwrap();
+    let req = test::TestRequest::post()
+        .uri("/indexes")
+        .insert_header(Encoder::Gzip.header().unwrap())
+        .insert_header((ACCEPT_ENCODING, "br"))
+        .insert_header(ContentType::json())
+        .set_payload(Encoder::Gzip.encode(body))
```

Comment:
That's funny! If I understand correctly, you are accepting answers encoded as brotli but you sent your request as _GZIP, and it works_!

Reply:
Yes. I outgoing request encoded with Gzip, and I ask response to be encoded with Brotli.